packages/astro/src/stores/internal.ts

```diff
@@ -16,10 +16,10 @@ export const $csrState = map<{
   organization: OrganizationResource | undefined | null;
 }>({
   isLoaded: false,
-  client: null,
-  user: null,
-  session: null,
-  organization: null,
+  client: undefined,
+  user: undefined,
+  session: undefined,
+  organization: undefined,
```

Comment:
See my comment [here](https://github.com/clerk/javascript/pull/3911/files#r1722623607). Putting the update here as they're related